### PR TITLE
Fix/1812 make sqlalchemy  url mandatory

### PR DESCRIPTION
**Pull request by inkhey**
related to #1812

## Checkpoints

- [ ] Code is clear enough
- [ ] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [ ] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [ ] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

### CODE REVIEW

File: backend/tracim_backend/migration/env.py [hunk 1]
```diff
@@ -21,6 +22,8 @@
 env_sqlalchemy_url = os.environ.get("TRACIM_SQLALCHEMY__URL")
 if env_sqlalchemy_url:
     config.set_main_option("sqlalchemy.url", env_sqlalchemy_url)
+if not config.get_main_option("sqlalchemy.url"):
+    raise ConfigurationError("SQLALCHEMY__URL is mandatory")
```
Comment (lebouquetin) [inline backend/tracim_backend/migration/env.py]: It seems not clear to me : you check `sqlalchemy.url` but you raise `SQLALCHEMY__URL` error. Do your code check the url from both config and envvar configuration?

Comment (inkhey) [reply backend/tracim_backend/migration/env.py]: Yes, it does it, sqlalchemy.url in config of alembic is at this point the correct value to check (there is no CFG here).
If you check the lines before we override config if there is env_var setted.

Comment (inkhey) [reply backend/tracim_backend/migration/env.py]: as SQLALCHEMY__URL is the parameter name of tracim that the best thing to return as mandatory.

**Comment by inkhey** [comment]
see #1920 for better fix for alembic context.

**Pull request merged**